src/data/scripts/internet/main.py

```diff
@@ -273,11 +258,12 @@ def read_jsonl_zst_files(dir_path):
         print("Finish processing")
 
         info = {"source": source, "current_version": version}
-        json.dump(info, open(f"{output_dir}/info.json", "w"))
+        create_info_file(info, output_dir)
 
         metadata = {
```

Comment:
Can we move the logic to create metadata dict into metadata.py to improve consistency across entire data pipeline